static/js/containers/ChannelPage.js

```diff
@@ -55,6 +56,7 @@ class ChannelPage extends React.Component {
               <ChannelSidebar channel={channel} />
             </Card>
           </div>
+          <br className="clear"/>
```

Comment:
Maybe I just don't understand what it's doing, but if I delete this element I don't see any changes in the display on the page. I'll leave it in for now if it's necessary.